lib/routes/changes.js

```diff
@@ -75,7 +75,14 @@ module.exports = function (app) {
                 console.log(err);
                 res.end();
                 cleanup();
+              }).on('complete', function () {
+                cleanup();
               });
+
+            req.connection.on('close', function () {
+              console.log('CLOSE!')
+              changes.cancel()
+            });
```

Comment:
Yup, just remove the `console.log()` and then I'm +1 on this. This seems reasonable to me.